recipe/meta.yaml

```diff
@@ -1,25 +1,17 @@
 {% set name = "maturin" %}
-{% set version = "1.3.1" %}
+{% set version = "1.5.1" %}
 
 package:
   name: {{ name }}-suite
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: efa194e99ae5fff185263d8244acacb12ae256ea73aba62c9446f6075ffc7ac1
-  patches:  # [s390x]
-    # ring adds big endian support from 0.17 only
-    - patches/0001-ring-0.17.3-for-s390x.patch # [s390x]
+  sha256: 3dd834ece80edb866af18cbd4635e0ecac40139c726428d5f1849ae154b26dca
 
 build:
   number: 0
-  skip: True  # [py<37]
-
-requirements: # [s390x]
-  build:      # [s390x]
-    - patch   # [s390x]
-
+  skip: True  # [py<37 or py>311]
```

Comment:
From what I can tell, the `python<3.11` requirement is only for the `tomli` dependency. See [here](https://github.com/PyO3/maturin/blob/1619479d9ef7ba35d68a3c347a899a8964155e6b/pyproject.toml#L15) and [here](https://github.com/PyO3/maturin/blob/1619479d9ef7ba35d68a3c347a899a8964155e6b/pyproject.toml#L8).

Reply:
Thanks for catching that! I totally misread [this line](https://github.com/PyO3/maturin/blob/1619479d9ef7ba35d68a3c347a899a8964155e6b/setup.py#L74) (I saw one too many semicolon).